integrations/docker/images/stage-2/chip-build-tizen/Dockerfile

```diff
@@ -35,6 +35,7 @@ RUN set -x \
 ENV TIZEN_VERSION 7.0
 ENV TIZEN_SDK_ROOT /opt/tizen-sdk
 
+# Fetch the version after tizen-unified_20230816.124244
```

Comment:
Please add todo and open a GH issue with link here, so we can read history and details.

Reply:
> Please add todo and open a GH issue with link here, so we can read history and details.

Done
Added TODO, and create GH issue
https://github.com/project-chip/connectedhomeip/issues/28852